buildSrc/src/main/groovy/net/minecraftforge/installer/tasks/Util.groovy

```diff
@@ -0,0 +1,157 @@
+package net.minecraftforge.installer.tasks
+
+import groovy.json.JsonSlurper
+
+import java.io.File
+import java.security.MessageDigest
+import java.text.SimpleDateFormat
+import java.util.Date
+
+import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.lib.AbbreviatedObjectId
+import org.eclipse.jgit.lib.ObjectId
+
+public class Util {
+	static void init() {
+		File.metaClass.sha1 = { ->
```

Comment:
You should get rid of all unused things in this class. See Forgeflower for its use.